e2e/cypress/test_module/cypress/support/step_definitions/header-steps.ts

```diff
@@ -0,0 +1,13 @@
+import { Given, Then, When } from "@badeball/cypress-cucumber-preprocessor";
+
+Given("the user is on home page", () => {
+  cy.visit("/");
+});
+
+When("the user views the header", () => {
+  cy.get(".header").should("be.visible");
+});
+
+Then("the user should see the header", () => {
+  cy.get(".header").should("be.visible");
+});

```

Comment:
the functionality of this step is exactly similar

Reply:
> the functionality of this step is exactly similar

Look at the description of this PR
It's just for testing work functionality it's enough

Reply:
> > the functionality of this step is exactly similar
> 
> Look at the description of this PR It's just for testing work functionality it's enough

But you also wrote that it's an example. I think the example must include best practices, to follow in the next PRs and also not create confusion

Reply:
@D1nviz In my case examples follow the logic of new package not writing the features with tests with best practices
FYI - I added some features with steps

Reply:
But they are  exactly similar lines

Reply:
> But they are exactly similar lines

I as I talked you:
In my case examples follow the logic for some tests of new package not writing the features with tests including best practices

You can create additional cases and show us.
I guess it'll be interesting to see best practices with your approaches